src/scenes/UserGalleryPage/QRCode.tsx

```diff
@@ -0,0 +1,207 @@
+import colors from 'components/core/colors';
+import styled from 'styled-components';
+import { useCallback, useState, useRef } from 'react';
+import Spacer from 'components/core/Spacer/Spacer';
+// import QRCodeStyling from 'qr-code-styling';
+
+function CloseIcon() {
+  return (
+    <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
it looks like we have a `CloseIcon` defined in `src/icons`, can we reuse that? 
I've been putting icons in that folder so it's easy to keep track of what icons we have